itests/src/test/resources/testconfiguration.properties

```diff
@@ -308,7 +308,6 @@ mr.query.files=\
   udf_count.q,\
   udf_using.q,\
   uniquejoin.q,\
-  vector_udf_trunc.q,\
```

Comment:
@ngsg, is that picked by LLAP driver? 
Also since MR support is removed, should we kill `mr.query.files` & `minimr.query.files` configs in another PR?

Reply:
@okumin, MapReduce compiler does not insert a TopNKey operator before the SELECT operator invoking trunc(). TNK operator flips batch.selectedInUse from false to true, so Tez passes batch.selectedInUse=true path while MR passes batch.selectedInUse=false path. This issue does not happen on the latter one.

Reply:
@deniskuzZ, Yes, now TestMiniLlapLocalCliDriver runs the test.
cf. http://ci.hive.apache.org/job/hive-precommit/job/PR-5300/1/artifact/test-results.tgz, split-19/itests/qtest/target/surefire-reports/TEST-org.apache.hadoop.hive.cli.split30.TestMiniLlapLocalCliDriver.xml

It will be nice to remove mr.query.files and minimr.query.files in new JIRA ticket. HIVE-23138 and its subtasks deal with this issue, but they seem outdated and deactivated.

Reply:
Sure thank. That makes sense